app/src/main/java/org/wikipedia/analytics/eventplatform/RecommendedReadingListEvent.kt

```diff
@@ -0,0 +1,36 @@
+package org.wikipedia.analytics.eventplatform
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import org.wikipedia.WikipediaApp
+import org.wikipedia.json.JsonUtil
+
+object RecommendedReadingListEvent {
+
+    fun submit(
+        action: String,
+        activeInterface: String,
+        groupName: String,
+        wikiId: String = WikipediaApp.instance.appOrSystemLanguageCode
+    ) {
+        val actionData = ActionData(
+            rrlGroup = groupName
+        )
+
+        EventPlatformClient.submit(
+            AppInteractionEvent(
+                action,
+                activeInterface,
+                JsonUtil.encodeToString(actionData).orEmpty(),
+                WikipediaApp.instance.languageState.appLanguageCode,
+                wikiId
```

Comment:
Please add a specific `streamName` to this event: 
```
 streamName = "app_rabbit_holes"
```
Reference: https://github.com/wikimedia/apps-android-wikipedia/pull/5114/files#diff-b94aed755cf060672bad411ea4686e073c58937e5e7a619e166f70e83f948b1a